### ISSUE TITLE
[HOLD]: Update breadcrumb current page to be a list item only, not a link

### ISSUE THREAD

**Issue by 1Copenut** [department-of-veterans-affairs/va.gov-team#516]
## Issue
The breadcrumbs currently uses an `<a>` with an aria-current attribute. This was flagged as a SC 4.1.2 violation. 

## Audit Finding
* Note 9, Defect 1

## Acceptance Criteria
* As a screen reader user, I should hear the `aria-current` as a list item instead of a link
* The breadcrumbs do not change visually

## Environment
* MacOS Mojave
* Chrome latest
* https://staging.va.gov/find-locations/

## Possible Fixes (optional)
The GOV.UK breadcrumb is similar in structure and shows a possible way to refactor our HTML.  https://design-system.service.gov.uk/components/breadcrumbs/ 

## WCAG or Vendor Guidance (optional)
* [Name, Role, Value: Understanding SC 4.1.2](https://www.w3.org/TR/UNDERSTANDING-WCAG20/ensure-compat-rsv.html)

**Comment by 1Copenut** [comment]
@alexpappasoddball Let's talk about this one before anyone starts work on it. I realized after I wrote it that the breadcrumb component uses some CSS to create a "back-by-one" breadcrumb at mobile sizes, so this change would break the default behavior on small screens.

**Comment by alexpappasoddball** [comment]
@1Copenut This has been assigned out to the facility locator team and is no longer in the triage backlog.  Not sure who should be assigned, but it looks like to be tagged appropriately now.  I'll go ahead and remove myself as the owner as it is no longer in our backlog and has been triaged out.

**Comment by 1Copenut** [comment]
I'm closing this issue after a discussion with the 508 group. We are in agreement the breadcrumb current page should have a link. No further action on this item is needed.
